moveit_ros/benchmarks/src/BenchmarkExecutor.cpp

```diff
@@ -889,6 +905,15 @@ void BenchmarkExecutor::collectMetrics(PlannerRunData& metrics,
       metrics["path_" + mp_res.description_[j] + "_clearance REAL"] = moveit::core::toString(clearance);
       metrics["path_" + mp_res.description_[j] + "_smoothness REAL"] = moveit::core::toString(smoothness);
       metrics["path_" + mp_res.description_[j] + "_time REAL"] = moveit::core::toString(mp_res.processing_time_[j]);
+
+      if (j == mp_res.trajectory_.size() - 1)
+      {
+        metrics["final_path_correct BOOLEAN"] = boost::lexical_cast<std::string>(correct);
```

Comment:
Should this be added instead of `path_x`? So you don't have duplicate metrics?

Reply:
All the other path metrics make totally sense since then you can compare certain planning steps within the same planner. I added this extra final_path just to compare the final result since OMPL, CHOMP, STOMP don't use the same naming pattern.